database/factories/FontFaceFactory.php

```
@@ -0,0 +1,59 @@
+			'axes'      => null,
+		];
+	}
+
+	/**
+	 * Attach variable-font axis metadata to the face.
+	 *
+	 * @since 1.7.0
+	 */
+	public function variable(): static
+	{
+		return $this->state( fn ( array $attributes ): array => [
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$attributes'. (undefined)

(UnusedFormalParameter)